frontend/helpers.js

```diff
@@ -31,16 +31,18 @@ export const getEmosproForUid = uid => ({
  * Returns the data for the ec_Event of a product or variant.
  * @param {Object} [baseProduct] The base (parent) product.
  * @param {Object} [variant] The selected variant.
+ * @param {boolean} sendParentUid The parent uid will be sent. Even for a variant article.
  * @return {Object} ec_Event data of the given product
  */
-export const getProductEventData = (baseProduct = {}, variant = {}) => {
+export const getProductEventData = (baseProduct = {}, variant = {}, sendParentUid = false) => {
   const variantAvailable = !!Object.keys(variant).length;
   const product = variantAvailable ? variant : baseProduct;
+  const productUid = (sendParentUid && variantAvailable) ? baseProduct.uid : product.uid;
 
   return {
     type: 'view',
     pid: baseProduct ? baseProduct.uid : '',
-    sku: variant.uid ? variant.uid : '',
+    sku: productUid,
     name: product.name,
     price: parseFloat(product.amount.net),
```

Comment:
i guess it should also take the price from the parent if `sendParentUid` is true?